src/app/modules/shared/shared.module.ts

```diff
@@ -0,0 +1,37 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { HeaderComponent } from './components/header/header.component';
+import { CommonModule } from '@angular/common';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
+import { RouterModule } from '@angular/router';
+
+@NgModule({
+    imports: [
+        CommonModule,
+        FormsModule,
+        NgbModule,
+        ReactiveFormsModule,
+        RouterModule
+    ],
+  
+    declarations: [
+      HeaderComponent
+    ],
+  
+    entryComponents: [
+    ],
+  
+    exports: [
+      HeaderComponent
+    ]
+  })
+  export class SharedModule {
+    static forRoot(): ModuleWithProviders {
+      return {
+        ngModule: SharedModule,
+        providers: [
+        ],
+      };
+    }
+  }
+  
```

Comment:
Whitespace!

Reply:
It's tslint rule
https://palantir.github.io/tslint/rules/whitespace/

Here why:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline